uint32 timestamp
uint8 E1
uint8 E2
uint8 lane_driving_C1
uint8 lane_driving_C2
uint8 lane_driving_C3
uint8 lane_driving_C4
uint8 lane_driving_E1
uint8 lane_driving_E2
uint8 lane_driving_F
uint8 lane_driving_state
uint8 upper_state

